src/App.tsx

```diff
@@ -68,6 +71,16 @@ export const App: React.FC = observer(() => {
               />
             }
           />
+          <Route
+            path="/take-quiz"
+            element={
+              <PrivateRoute
+                {...defaultProtectedRouteProps}
```

Comment:
I would recommend moving the `defaultProtectedRouteProps` in the `PrivateRoute` component as it makes more sense having default values already inside the component, thus not having the need to always pass them as props.